### PR TITLE
fix: redirect to email sign up on first login

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
I'm using `acceptedPrivacyTerms` as a proxy for 'has logged in before'.  In reality new users have to have accepted the terms before logging in, but it's a useful flag that can be used to ask them (once) if they would like the newsletter.  I think we should migrate to a better name, though.

Also, this breaks the redirect via query parameters. Sorry about that - I'll try to restore it later.

### CODE REVIEW

File: client/src/pages/email-sign-up.js [hunk 1]
```diff
@@ -55,74 +53,43 @@ class AcceptPrivacyTerms extends Component {
...
+    const { quincyEmail } = this.state;
     return (
       <Fragment>
         <Helmet>
-          <title>Privacy Policy and Terms of Service | freeCodeCamp.org</title>
+          <title>Email Sign Up | freeCodeCamp.org</title>
         </Helmet>
         <Spacer size={2} />
         <Row className='text-center'>
           <Col sm={8} smOffset={2} xs={12}>
```
Comment (raisedadead) [inline client/src/pages/email-sign-up.js:85]: These do not look right to me. Or, I might be missing something. Either way @ahmadabdolsaheb can you check the grid layout is the correct on this page before I test this.

**Comment by ahmadabdolsaheb** [comment]
@raisedadead, Updated the UI. 
image

**Comment by raisedadead** [comment]
Thanks, the UI looks better, but we should use a native check box instead of this radio button (atleast thats what it appears like to the unsuspecting user).

**Comment by ahmadabdolsaheb** [comment]
Screen Shot 2020-08-04 at 11 12 18 PM

**Pull request merged**